src/main/java/core/basesyntax/CompareTwoAuto.java

```diff
@@ -1,9 +1,15 @@
 package core.basesyntax;
 
+import java.util.function.BinaryOperator;
+
 /**
  * <p>Имплементируйте необходимый функциональный интерфейс и его метод для сравнения мощности
  * двигателей у двух авто (enginePower) и возврата разницы между ними.
  * Метод должен принимать Integer.</p>
  */
-public class CompareTwoAuto {
+public class CompareTwoAuto implements BinaryOperator<Integer> {
+    @Override
+    public Integer apply(Integer integer, Integer integer2) {
+        return integer - integer2;
```

Comment:
https://mate-academy.github.io/jv-program-fulltime/code_style_requirements.html

read the whole document and pay attention to the very first point